### PR TITLE
Update Minetweaker.zs

### PR DESCRIPTION
**Pull request by hilburn**

### CODE REVIEW

File: scripts/Minetweaker.zs [hunk 1]
```diff
@@ -56,4 +57,4 @@ val Au = <ore:element_Au>;
 //Element Types = "Non-metal","Inert gas","Halogen","Alkali metal","Alkaline Earth Metal","Metalloid","Other metal",
 //				  "Transition metal","Lanthanide","Actinide";
 //Room States   = "Liquid","Solid","Gaseous";
-//Radioactivity = "Stable","Hardly Radioactive","Slightly Radioactive","Radioactive","Highly Radioactive","Extremely Radioactive";
\ No newline at end of file
+//Radioactivity = "Stable","Hardly Radioactive","Slightly Radioactive","Radioactive","Highly Radioactive","Extremely Radioactive";
```
Comment (jakimfett) [inline scripts/Minetweaker.zs]: Was this just a change to line endings?

Comment (hilburn) [inline scripts/Minetweaker.zs]: errr... not entirely sure why that's there, but no - i added in the MT val water = <fluid:water>; as it was stopping the example fluid decomp scripts from working

Comment (jakimfett) [inline scripts/Minetweaker.zs]: Gotcha.

**Comment by jakimfett** [comment]
Thanks for fixing this!

**Pull request merged**